script.js

```diff
@@ -16,26 +23,55 @@ restaurantsApp.getRestaurantIDs = (searchTerm, searchLocation) => {
         },
         data: {
             term: searchTerm,
-            location: searchLocation
+            location: searchLocation,
+            limit: 3,
         }
     }).then(function (result) {
-        //This takes the first three results, can be changed later
-        for (let i = 0; i < 3; i++) {
-            let businessID = result.businesses[i].id
-            let businessName = result.businesses[i].name
-            let businessRating = result.businesses[i].rating
-            let businessPrice = result.businesses[i].price
-            let businessAddress = result.businesses[i].location.display_address
-        }
-    });
+        console.log(result)
+        // result.bussinesses is an array on bussinesses that can be looped through with forEach method ,then for each business object the info are stored in respective array
+        result.businesses.forEach(function(item){
+
+             restaurantsApp.businessID.push(item.id)
+             restaurantsApp.businessRating.push(item.rating)
+             restaurantsApp.businessPrice.push(item.price)
+             restaurantsApp.businessName.push(item.name)
+             restaurantsApp.businessAddress.push(item.location.display_address) 
+          
+    }); 
+    restaurantsApp.getReviews()
+})
+
+}
+restaurantsApp.getReviews=function(){
```

Comment:
can you change this to accept a single `businessId` parameter?  then instead of looping over all the IDs in the `retaurantApp.businessID` array , just make a call for the single `businessId`?

the reason for this is that we're going to need the reviews only when the user hovers over the image.

Reply:
I figured we can make the call and store the data in an array to be displayed when the user hovers. if we make the call when the user hovers its gonna take a while for results to come back and to be displayed which would not make a very pleasant user experience.

Reply:
based on our discussions we should change this line to:

```
restaurantsApp.getReviews=function(businessIDs){
```